### ISSUE TITLE
bug(YouTube/Hide comment component): `Hide emoji picker` hides timestamps for chapters

### ISSUE THREAD

**Issue by jthead9** [inotia00/ReVanced_Extended#1556]
### Type

Error at runtime

### Tools used

ReVanced Manager

### Application

Revanced Extended 18.37.36

### Bug description

When I'm watching a youtube video, I normnally pull up the chapters on mobile. But when I do, only the active chapter has time stamps. No other chapter does. 

### Steps to reproduce

Step 1) Open the app
Step 2) Open a video with chapters in it
Step 3) Turn phone landscape* (Optional as it does it without landscape as well)
Step 4) Click on active Chapters
Step 5) Look at chapters

### Relevant log output

```shell
I don't have any. If I do need to provide, please let me know.
```

### Screenshots or videos

![xx5oykx5qtub1](https://github.com/inotia00/ReVanced_Extended/assets/58606862/664f8053-db40-4cb5-b91c-6e7b61dc1b3d)

### Acknowledgements

- [x] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] I have written the title and contents in English.
- [X] This issue does not reproduce on unpatched YouTube or YT Music.

**Comment by KobeW50** [comment]
For what it's worth, I tried excluding the Hide Layout Components patch, and the Hide Description Components patch but the issue still appears. Hope that helps in any small way.

**Comment by jthead9** [comment]
I tried this but no luck.

**Comment by inotia00** [comment]
This issue will be fixed in the next release
Temporarily turn off `Hide Emoji Picker` until the next patch rolls out

**Comment by inotia00** [comment]
fixed in [revanced-patches-v2.193.18](https://github.com/inotia00/revanced-patches/releases/tag/v2.193.18)
